.spectral.yaml

```diff
@@ -42,6 +42,27 @@ rules:
     then:
       field: x-options
       function: truthy
+  x-alias-schema:
+    description: x-alias must have the correct schema.
+    message: "{{description}} {{error}}"
+    given: "$.paths[*][get,put,post,delete,options,head,patch,trace].x-alias"
+    severity: error
+    then:
+      function: schema
+      functionOptions:
+        schema:
+          type: string
+          pattern: "^[a-z]+[a-z-]?[a-z]+$"
```

Comment:
that's an odd regex.  if you want at most one hyphen, use `^[a-z]+-?[a-z]+$`.  If you want to allow multiple, but not at start or end, use `^[a-z][a-z-]*[a-z]`.

Reply:
That is what I meant. Thanks :)